Sources/Vapor/HTTP/Server/HTTPServer.swift

```diff
@@ -388,7 +404,8 @@ private final class HTTPServerConnection: Sendable {
     ) -> EventLoopFuture<HTTPServerConnection> {
         let quiesce = ServerQuiescingHelper(group: eventLoopGroup)
         let bootstrap = ServerBootstrap(group: eventLoopGroup)
-            /// Specify backlog and enable `SO_REUSEADDR` for the server itself.
+            /// Specify accepts per loop and backlog, and enable `SO_REUSEADDR` for the server itself.
+            .serverChannelOption(ChannelOptions.maxMessagesPerRead, value: UInt(configuration.requestsPerServerTick))
```

Comment:
Is the conversion needed?!

```suggestion
            .serverChannelOption(ChannelOptions.maxMessagesPerRead, value: configuration.requestsPerServerTick)
```

Reply:
Nope, good catch!